src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -141,16 +149,31 @@ public static String getIndividualLastName(Entity entity) {
     }
 
     public static String getIndividualPreferredHearingChannel(String hearingType, HearingSubtype hearingSubtype) {
-        // TODO Depends on SSCS-10273 - Needs to implement for Reference data of valid Hearing Channel codes
+        if (hearingType != null || hearingSubtype != null) {
+            if (HEARING_TYPE_PAPER.equals(hearingType)) {
+                return NOT_ATTENDING.getHmcReference();
+            }
+
+            if (isYes(hearingSubtype.getWantsHearingTypeFaceToFace())) {
+                return FACE_TO_FACE.getHmcReference();
+            } else if (isYes(hearingSubtype.getWantsHearingTypeVideo())) {
+                return VIDEO.getHmcReference();
+            } else if (isYes(hearingSubtype.getWantsHearingTypeTelephone())) {
+                return TELEPHONE.getHmcReference();
+            }
+        }
         return null;
     }
 
     public static String getIndividualInterpreterLanguage(HearingOptions hearingOptions) {
-        // TODO Depends on SSCS-10273 - Needs to implement for Reference data to convert from SSCS Languages/Sign Languages to Reference languages
-        // if (isYes(hearingOptions.getLanguageInterpreter())) {
-        //     String signLanguageType = hearingOptions.getSignLanguageType();
-        //     String languages = hearingOptions.getLanguages();
-        // }
+        if (hearingOptions.wantsSignLanguageInterpreter()) {
+            String signLanguageType = hearingOptions.getSignLanguageType();
+            return SignLanguage.getSignLanguageKeyByCcdReference(signLanguageType).getHmcReference();
```

Comment:
what happens if `SignLanguage.getSignLanguageKeyByCcdReference(signLanguageType)`  returns null ? We should guard against it.